.github/workflows/tests.yml

```diff
@@ -1,24 +1,24 @@
+  
 name: Tests
 
-on:
-  push:
-    branches: [ master ]
-  pull_request:
-    branches: [ master ]
+on: [push, pull_request]
 
 jobs:
-  build:
-
+  run:
     runs-on: ubuntu-latest
 
-    steps:
-      - uses: actions/checkout@v2
-
-      - name: Validate composer.json and composer.lock
-        run: composer validate
+    strategy:
+      matrix:
+        php: [7.1, 7.2, 7.3, 7.4]
```

Comment:
is there a "nightly" or "7" version?

Reply:
There is a [experimental setup](https://github.com/shivammathur/setup-php#experimental-setup). I'm don't know how this could be added without failing the tests.

Reply:
:+1: